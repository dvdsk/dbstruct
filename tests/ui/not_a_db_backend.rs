use dbstruct::dbstruct;

#[dbstruct(db=starship_voyager)]
struct Test {
    field: Option<u8>,
}

fn main() {}
