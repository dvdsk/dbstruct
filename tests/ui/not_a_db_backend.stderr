error: Not a known database backend: `starship_voyager`
 --> tests/ui/not_a_db_backend.rs:3:15
  |
3 | #[dbstruct(db=starship_voyager)]
  |               ^^^^^^^^^^^^^^^^
  |
  = help: try sled as database backend
